catalog/plugin/catalog-plugin-metacardbackup/src/main/java/org/codice/ddf/catalog/plugin/metacard/backup/MetacardBackupPlugin.java

```diff
@@ -180,7 +182,8 @@ private void processRequest(ProcessRequest<? extends ProcessResourceItem> proces
             Metacard metacard = processResourceItem.getMetacard();
             try {
                 LOGGER.trace("Backing up metacard : {}", metacard.getId());
-                BinaryContent binaryContent = metacardTransformer.transform(metacard, null);
+                BinaryContent binaryContent = metacardTransformer.transform(metacard,
+                        Collections.emptyMap());
```

Comment:
Added this change to address CID 161296 (a non-issue in fact, but simple enough to silence here).